packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -648,9 +657,8 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
     action,
     () => ({
       updateIndicator: updateIndicatorState,
-      updateScrollButtons: updateScrollButtonState,
```

Comment:
This is a breaking change. Devs could be relying on the `updateScrollButtons` returned by the `action`. See the `prop: action` test case.

Reply:
Good point, i added it as deprecated

Reply:
I believe we mark `@deprecated` when we have an alternative solution. In this case, we should keep this method and only remove passing it from `onScroll` in `TabsScroller`. This way, internally, it will utilize the IntersectionObserver API, and `updateScrollButton`s will remain accessible for developers who are using it.

Reply:
Done :wink: